src/cl-giskard.lisp

```diff
@@ -50,58 +50,67 @@
 (defparameter *giskard-action-goal-type* "giskard_msgs/WholeBodyCommand")
 
 (defparameter *tf-listener* nil)
+(defparameter *tf-default-timeout* 2.0)
 
 (defun ensure-tf-listener ()
-  (if *tf-listener*
-    *tf-listener*
-    (progn
-      (setf *tf-listener* (make-instance 'cl-tf:transform-listener))
-      (roslisp:wait-duration 1.0)
-      *tf-listener*)))
-
-(defun ps->msg (pose-stamped)
-  (cl-transforms-stamped:make-pose-stamped-msg (cl-transforms-stamped:pose-stamped->pose pose-stamped)
-                                               (cl-transforms-stamped:frame-id pose-stamped)
-                                               (cl-transforms-stamped:stamp pose-stamped)))
+  (unless *tf-listener*
```

Comment:
I'm not sure whether I can get away with not waiting for the tf-listener here. Other ROS objects (topic publishers, actionlib clients) need a bit of time to set themselves up after a make-instance, and I expect a tf-listener to be the same.